apps/system-e2e/src/addons.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, Locator } from '@playwright/test'
+
+const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
+
+expect.extend({
+  async toHaveCountGreaterThan(
+    received: Locator,
+    value: number,
+    options?: { timeoutMs?: number },
+  ) {
+    let count = 0
+    const oneSecondMs = 1000
+    const maxRetries = 60
+    for (let i = 0; i < maxRetries; i++) {
+      count = await received.count()
```

Comment:
Avoid C-style loops

Reply:
It does perfectly the job in this case 🤷🏻‍♂️ .